viaacode/hetarchief-client
# Feature/arc 1285

https://meemoo.atlassian.net/browse/ARC-1285

How to test: 
1. In `hetarchief-proxy`, change the `REROUTE_EMAILS_TO` env variable value to your own email
3. Go to localhost:3200/nieuwsbrief
4. If you're logged in you should be redirected to your profile
5. If you're logged out you should see the newsletter form
6. Fill in the form + submit
7. You'll receive an email
8. Click the confirm button
9. You should be redirected to the homepage and see a toast message

https://user-images.githubusercontent.com/77958553/236852550-893398fa-7af1-41bc-a895-2024c25cd23b.mov
